### DISCUSSION TITLE
Missing the  "ruleset" field  in the payload send to engine.

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#28] [Bug Reports]
The payload send to engine : 
`"game": {
		"id": "4f4d39cc-1136-4334-a404-914f816691d1",
		"timeout": 500
	},`

Missing the ruleset field

**Comment by Nettogrof** [comment]
This will be resolved by pull BattlesnakeOfficial/rules#32

**Reply by andrewdmaclean** [reply]
Hey @Nettogrof 👋 Just wanted to thank you for this PR - I'm just pinging @aurorawalker to pop it up to the top of her notifications - not sure where things are sitting right now.

**Comment by bvanvugt** [comment]
Created Issue https://github.com/BattlesnakeOfficial/feedback/issues/65 to address this. Thanks @Nettogrof!

**Comment by bvanvugt** [comment]
Should be fixed in [Rules v1.0.18](https://github.com/BattlesnakeOfficial/rules/releases/tag/v1.0.18)

(thanks @joram for having already opened a PR for this!)
